doc/Language/rb-nutshell.pod6

```diff
@@ -263,7 +263,7 @@ say %calories« $keys ».join(','); # Perl 6, interpolated split
 I<Key/value-slicing>
 
 =for code :lang<ruby>
-say calories.slice('pear', 'plum').join(','); # Ruby, with ActiveRecord
+puts calories.slice('pear', 'plum')
```

Comment:
`#slice` returns a `Hash`, with key value pairs for `'pear'` and `'plum'` so calling `#join` results in an error. 

Not sure what the Perl 6 equivalent is –or if there is one. Let me know and I will make the appropriate updates.

Reply:
Simply using the indexing operator <>: calories<pear plum>

Reply:
I don't follow, sorry. 

```
~ >ruby -e'p ({a:1, b:2, c:3}).slice(:a, :c)'
{:a=>1, :c=>3}
```

To get that, seems one must do:

```~ >perl6 -e'my %h = (a => 1, b => 2, c => 3); say %h<a c>:kv'
(a 1 c 3)
```

But, since that's a `List`, one must assign to a `Hash`: `my %h2 = %h<a c>:kv'`. Maybe there's a way without assigning the list to `%h2`? Maybe there's a way without having to add `:kv` which, if you're coming to Perl 6 from Ruby may not be a good selling point.

Reply:
Add `.Hash`:

    perl6 -e'my %h = (a => 1, b => 2, c => 3); say %h<a c>:kv.Hash'

Reply:
Great, thanks! PR updated. 🦋